cmd/processor/app/processor.go

```diff
@@ -158,6 +159,11 @@ func NewProcessor(configurationPath string, platformConfigurationPath string) (*
 		return nil, errors.Wrap(err, "Failed to create and start health check server")
 	}
 
+	if (processorConfiguration.Spec.DisableDefaultHttpTrigger == nil || *processorConfiguration.Spec.DisableDefaultHttpTrigger) &&
+		len(functionconfig.GetTriggersByKind(processorConfiguration.Spec.Triggers, "http")) > 0 {
+		startInternalHealthCheck()
```

Comment:
that needs to respond with "ok" only after all triggers were successfully created or else the function pod will report readiness while triggers are not ready to receive requests